### PR TITLE
Feature/navigation

### PR DESCRIPTION
**Pull request by ppatilgit**
Navigation ready for code review

### CODE REVIEW

File: src/containers/UserDashboard/UserDashboard.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        <div className="userdashboard_main_top">
+          <div className="searchbox">
+            <img src={searchImg} alt="Search" />
+            <input
+              type="text"
+              name="test"
+              id="search"
+              value={searchTerm}
+              onInput={handleInput}
+              defaultValue="Search.."
```
Comment (tinasam) [inline src/containers/UserDashboard/UserDashboard.jsx:34]: Please show clear "Search" on click

**Review by tinasam — COMMENTED** [review]
see comments on userDashboard

Looking forward to all routing pages being connected.

Great job!

File: src/containers/UserDashboard/UserDashboard.jsx (hunk 1 above)
Comment (ppatilgit) [reply src/containers/UserDashboard/UserDashboard.jsx]: Updated defaultValue attribute to placeHolder, and it is fixed

**Review by tinasam — APPROVED** [review]
Reviewed update of placeholder.

**Pull request merged**